src/Interest.hs

```diff
@@ -63,6 +63,7 @@ data InterestDatum = InterestDatum
     , liquidationCommission :: !Integer   -- How much % borrower will pay for lender when liquidated (before time passes)
     , requestExpiration     :: !POSIXTime
     , lenderNftTn           :: !TokenName
+    , lendDate              :: !POSIXTime
```

Comment:
I guess `InterestDatum` does not need this and also the majority of the other fields as well 🤔

Reply:
resolved in other branch